src/sentry/integrations/pipeline.py

```diff
@@ -43,17 +43,26 @@ def ensure_integration(key, data):
 
 
 class IntegrationPipeline(Pipeline):
+    PREEXISTING_INTEGRATION = 'preexisting_integration'
```

Comment:
Instead of using a constant here, can you actually add a comment into the `sentry.integrations.base.IntegrationProvider`'s `build_integration` docstring noting that you can pass `expect_exists`, which would indicate that instead of attempting to create and update the integration, just expect that the passed external_id exists and do not attempt to create or update the integration.